sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/details/key_constants.hpp

```diff
@@ -58,4 +58,15 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
   constexpr static const char DeletedOnPropertyName[] = "deletedDate";
   constexpr static const char ScheduledPurgeDatePropertyName[] = "scheduledPurgeDate";
 
+  /***************** Curve Names *****************/
+  constexpr static const char P256Value[] = "P-256";
+  constexpr static const char P256KValue[] = "P-256K";
```

Comment:
Just a note: this has a different curve name on macOS, and may not be supported on older mac and Windows versions. Just something to keep in mind. In .NET, I had to figure out the OID for macOS so you can look there.